ml_service/pipelines/employer_engagement_build_pipeline.py

```diff
@@ -0,0 +1,82 @@
+import azureml.core
+import os
+from azureml.core import Workspace, Datastore, Dataset, ComputeTarget, Experiment, ScriptRunConfig, Environment, Model
+from azureml.core.run import Run
+from azureml.core.runconfig import RunConfiguration
+from azureml.pipeline.steps import PythonScriptStep
+from azureml.pipeline.core import Pipeline, PipelineData, StepSequence, PublishedPipeline
+from ml_service.util.attach_compute import get_compute
+from ml_service.util.env_variables import Env
+from ml_service.util.manage_environment import get_environment
+import os
+
+# Set up all environment details
+e = Env()
+subscription_id = os.environ.get("SUBSCRIPTION_ID")
+workspace_name = os.environ.get("WORKSPACE_NAME")
+resource_group = os.environ.get("RESOURCE_GROUP")
+
+# Get Azure machine learning workspace
+aml_workspace = Workspace.get(
+name=e.workspace_name,
+subscription_id=e.subscription_id,
+resource_group=e.resource_group,
+)
+print("get_workspace:")
+print(aml_workspace)
+
+# Create experiment if it doesn't exist
+experiment = Experiment(aml_workspace, "employer-engagement")
+
+# Add compute information to variable
+aml_compute = aml_workspace.compute_targets["cpucluster"]
+
+# Set up experiment folder
+experiment_folder = 'employer-engagement'
+
+# Create a Python environment for the experiment (from a .yml file)
+environment = Environment.from_conda_specification("environment", experiment_folder + "/conda_dependencies.yml")
+# Register the environment 
+environment.register(workspace=aml_workspace)
+registered_env = Environment.get(aml_workspace, 'environment')
+# Create a new runconfig object for the pipeline
+pipeline_run_config = RunConfiguration()
+# Use the compute you created above. 
+pipeline_run_config.target = aml_compute
+# Assign the environment to the run configuration
+pipeline_run_config.environment = registered_env
+print ("Run configuration created.")
+
+# Use train step to build and model
+# train_source_dir="./employer-engagement/training"
+# train_step = PythonScriptStep(
+#     name='model_build',
+#     script_name="train.py",
+#     compute_target=aml_compute,
+#     runconfig=pipeline_run_config,
+#     source_directory=train_source_dir)
```

Comment:
Can we pull this out into a separate training pipeline please

Reply:
Pipelines separated